lib/os/thread_posix.cpp

```diff
@@ -45,9 +47,7 @@
 namespace pstore {
     namespace threads {
 
-#    ifdef __FreeBSD__
         static thread_local char thread_name[name_size];
```

Comment:
Don’t define this if we don’t need it. (An unused static will likely trigger a compiler warning.)

Reply:
Thanks, fixed in the commit https://github.com/SNSystems/pstore/pull/93/commits/88195a24bcbd4e6e349f08bc63cdc1197b10f153.